blaze/compatibility.py

```diff
@@ -133,3 +136,17 @@ def assert_series_equal(left, right, check_names=True, **kwargs):
         if check_names:
             assert left.name == right.name
         return tm.assert_series_equal(left, right, **kwargs)
+
+
+def _pandas_sort_old(df, *args, **kwargs):
+    """Use pd.sort() for older versions"""
+    return df.sort(*args, **kwargs)
+
+def _pandas_sort_new(df, *args, **kwargs):
+    """Use pd.sort_values() for newer versions"""
+    return df.sort_values(*args, **kwargs)
+
+if LooseVersion(pd.__version__) < '0.17.0':
+    pandas_sort = _pandas_sort_old
+else:
+    pandas_sort = _pandas_sort_new
```

Comment:
You should define this in `compute/pandas.py` since it's only relevant there. `compatibility` is for language compatibility